release_notes/ocp-4-5-release-notes.adoc

```diff
@@ -274,10 +273,86 @@ deprecated in {product-title} 3.11 and has been removed. You must use the
 deprecated in {product-title} 3.11 and have been removed. You must use the
 `--suggest` flag with this command instead.
 
+[id="ocp-4-5-service-catalog-removed"]
+==== Service Catalog, Template Service Broker, and their Operators
+
+[IMPORTANT]
+====
+Service Catalog is not installed by default in {product-title} 4, however it now
+blocks upgrades to {product-title} 4.5 if installed.
+====
+
+Service Catalog, Template Service Broker, Ansible Service Broker, and their
+associated Operators were deprecated starting in {product-title} 4.2. Ansible
+Service Broker, including Ansible Service Broker Operator and related APIs and
+APBs, were removed in {product-title} 4.4.
+
+Service Catalog, Template Service Broker, and their associated Operators are now
+removed in {product-title} 4.5, including the following related API:
+
+* `.servicecatalog.k8s.io/v1beta1`
+
+[NOTE]
+====
+Templates are still available in {product-title} 4.5, but they are no longer
+handled by Template Service Broker. By default, the Samples Operator handles Red
+Hat Enterprise Linux (RHEL)-based {product-title} ImageStreams and Templates.
+See
+xref:../openshift_images/configuring-samples-operator.adoc#configuring-samples-operator[Configuring the Samples Operator]
+for details.
+====
+
+The `service-catalog-controller-manager` and `service-catalog-apiserver` cluster
+Operators were set to `Upgradeable=false` in 4.4. This means that they block
+cluster upgrades to the next minor version, 4.5 in this case, if they are still
+installed at that time. Upgrading to z-stream releases such as 4.4.z, however,
```

Comment:
Small nit: it's slightly ambiguous how this could read as as either "upgrading...is permitted" or "upgrading releases are permitted". Suggest changing _Upgrading_ to _Upgrades_.